foundation/foundation-mda/src/main/java/com/boozallen/aiops/mda/generator/VersioningDockerModuleGenerator.java

```diff
@@ -65,12 +65,6 @@ public void generate(GenerationContext context) {
             final String appName = "versioning";
 
             manualActionNotificationService.addNoticeToAddModuleToParentBuild(context, artifactId, "docker");
-            DockerBuildParams params = new DockerBuildParams.ParamBuilder()
```

Comment:
A: We should probably delete `DockerBuildParams` entirely.

Reply:
Yeah sounds good.. will do that when removing docker_build() for both inference and training pipeline